client/my-sites/sidebar/sidebar.jsx

```diff
@@ -166,18 +166,35 @@ module.exports = React.createClass( {
 			themesLink = '/design';
 		}
 
-		return (
-			<SidebarItem
-				tipTarget="themes"
-				label={ this.translate( 'Themes' ) }
-				className={ this.itemLinkClass( '/design', 'themes' ) }
-				link={ themesLink }
-				buttonLink={ getCustomizeUrl( null, site ) }
-				buttonLabel={ this.translate( 'Customize' ) }
-				onNavigate={ this.onNavigate }
-				icon={ 'themes' }
-				preloadSectionName="themes" />
-		);
+        var abtest = require( 'lib/abtest' ).abtest;
+        
+		if ( abtest( 'swapButtonsMySiteSidebar' ) === 'swap' ) {
+		    return (
+		        <SidebarItem
+				    tipTarget="themes"
+				    label={ this.translate( 'Customize' ) }
+				    className={ this.itemLinkClass( '/design', 'themes' ) }
+				    link={ getCustomizeUrl( null, site ) }
+				    buttonLink={ themesLink }
+				    buttonLabel={ this.translate( 'Themes' ) }
+				    onNavigate={ this.onNavigate }
+				    icon={ 'themes' }
+				    preloadSectionName="themes" />
+		    );
+    	} else {
```

Comment:
Since each of these blocks is a `return` statement, you don't really need the `else`. If the first one is triggered, it will return and the second one will fire instead.